Source/WebKit/UIProcess/API/gtk/WebKitWebViewBasePrivate.h

```diff
@@ -52,7 +52,7 @@ void webkitWebViewBaseSetTooltipText(WebKitWebViewBase*, const char*);
 void webkitWebViewBaseSetTooltipArea(WebKitWebViewBase*, const WebCore::IntRect&);
 void webkitWebViewBaseSetMouseIsOverScrollbar(WebKitWebViewBase*, WebKit::WebHitTestResultData::IsScrollbar);
 void webkitWebViewBaseForwardNextKeyEvent(WebKitWebViewBase*);
-void webkitWebViewBaseForwardNextWheelEvent(WebKitWebViewBase*);
+void webkitWebViewBaseForwardNextWheelEvent(WebKitWebViewBase*, GdkEvent*);
```

Comment:
This should probably be renamed to `webkitWebViewBaseForwardWheelEvent`

Reply:
Done